bindings/python/src/OpenSpaceToolkitPhysicsPy/Environment/Gravitational/Earth.cpp

```diff
@@ -40,7 +40,7 @@ inline void                     OpenSpaceToolkitPhysicsPy_Environment_Gravitatio
 
         enum_<Earth::Type>(earth_class, "EarthType")
 
-            .value("spherical", Earth::Type::Spherical)
+            .value("Spherical", Earth::Type::Spherical)
```

Comment:
Agreed, @apaletta3 we had discussed this, nah?

Reply:
yes we had, sounds good